apps/dashboard/src/lib/utils/types/index.ts

```diff
@@ -391,3 +391,27 @@ interface Tabs {
   id: number;
   name: string;
 }
+
+export interface CollectionCourse {
+  id: string;
+  avatar: string;
+  title: string;
+  description: string;
+  lessonNumber: number;
+  studentNumber: number;
+  is_unlocked: boolean;
+  is_completed: boolean;
+  is_published: boolean;
+}
+
+export interface Collection {
+  slug: string;
+  title: string;
+  avatar: string;
+  description: string;
+  prerequisite: string,
+  is_published: boolean,
+  lms_certificate: boolean,
+  courses_certificate: string,
+  courses: CollectionCourse[]
```

Comment:
Please never use Collection to avoid confusion, replace all word `Collection` to `Pathway` for consistency.